src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,19 @@
+import {useAuth0} from "@auth0/auth0-react";
+import {Button} from "@mui/material";
+import "./Dasboard.css";
+
+function Dashboard() {
+  const {logout, user, isAuthenticated} = useAuth0();
+  return (
+    isAuthenticated &&
+      <div className="dashboard">
```

Comment:
Redirect to the login page if user is not authenticated

Reply:
This page is just for demo purpose, lets have the another story for the protection of the route? what do you say?